components/hdw-tft/hdw-tft.c

```diff
@@ -448,7 +491,9 @@ void initTFT(display_t * disp, spi_host_device_t spiHost, gpio_num_t sclk,
 
 #if defined(CONFIG_GC9307_240x280)
     esp_lcd_panel_invert_color(panel_handle, false);
-    esp_lcd_panel_io_tx_param(io, 0x36, (uint8_t[]) {0x28}, 1 ); //MX, MY, RGB mode  (MADCTL)
+    // NOTE: the following call would override settings set by esp_lcd_panel_swap_xy() and esp_lcd_panel_mirror()
+    // Both of the prior functions write to the 0x36 register
+    // esp_lcd_panel_io_tx_param(io, 0x36, (uint8_t[]) {0xE8}, 1 ); //MX, MY, RGB mode  (MADCTL)
```

Comment:
@cnlohr This call was clobbering prior values. Can you make sure that me commenting it out didn't break anything horrible? If it did, then move these functions after the register writes
```c
esp_lcd_panel_swap_xy(panel_handle, SWAP_XY);
esp_lcd_panel_mirror(panel_handle, MIRROR_X, MIRROR_Y);
esp_lcd_panel_set_gap(panel_handle, X_OFFSET, Y_OFFSET);
```